SPECS/kernel/config

```diff
@@ -2119,6 +2119,7 @@ CONFIG_ZRAM_DEF_COMP_LZORLE=y
 # CONFIG_ZRAM_DEF_COMP_LZO is not set
 CONFIG_ZRAM_DEF_COMP="lzo-rle"
 # CONFIG_ZRAM_WRITEBACK is not set
+# CONFIG_ZRAM_TRACK_ENTRY_ACTIME is not set
```

Comment:
Result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/b7c3fd65a3eaa7a7639ddec8b817a824e814bf37

This is a new option is selected if `ZRAM_MEMORY_TRACKING` is enabled which is on in ARM64 but not in x86_64